app/services/platforms/twitch/streams.ts

```diff
@@ -0,0 +1,30 @@
+import { Observable } from 'rxjs/Observable';
+import { ajax } from 'rxjs/observable/dom/ajax';
+import { map } from 'rxjs/operator/map';
+import { TwitchPagination } from './pagination';
+import { TwitchRequestHeaders } from './tags';
+import 'rxjs/add/operator/map';
+
+export type Stream = {
+  id: string;
+  user_id: string;
+  user_name: string;
+  game_id: string;
+  community_ids: string[];
+  type: string;
+  title: string;
+  viewer_count: number;
+  started_at: string;
+  language: string;
+  thumbnail_url: string;
+};
+
+export type StreamsResponse = {
+  data: Array<Stream>;
+  pagination: TwitchPagination;
+};
+
+export const getUserStreams = (userId: string, headers: TwitchRequestHeaders) =>
+  ajax
+    .getJSON<StreamsResponse>(`https://api.twitch.tv/helix/streams?user_id=${userId}`, headers)
+    .map(response => response.data);
```

Comment:
same as above, i feel like this function could be in the class file (if not a part of the class) and the interfaces could go in an api file

Reply:
Would love to hear your thoughts on this, if we do add it to the same file then we're likely adding it after the class and then we'll have "use before define" warnings. Above the class would pollute it. 
As for the types in an API file, isn't that the same that having a separate file as is now (I couldn't find any, apologies if there's is one).